src/components/app/flyouts/settings/settings.js

```diff
@@ -99,13 +118,35 @@ export class Settings extends LinkedComponent {
   };
 
   render() {
-    const { t, onClose, theme, changeTheme, version, releaseNotesUrl, isSimulationEnabled,
-      simulationToggleError, setLogoError, getSimulationPending, getSimulationError } = this.props;
+    const {
+      t,
+      onClose,
+      theme,
+      changeTheme,
+      version,
+      releaseNotesUrl,
+      isSimulationEnabled,
+      simulationToggleError,
+      setLogoError,
+      getSimulationPending,
+      getSimulationError
+    } = this.props;
     const nextTheme = theme === 'dark' ? 'light' : 'dark';
-    const { desiredSimulationState, loading, logoFile, applicationName, toggledSimulation, madeLogoUpdate } = this.state;
+    const {
+      desiredSimulationState,
+      loading,
+      logoFile,
+      applicationName,
+      toggledSimulation,
+      madeLogoUpdate
+    } = this.state;
     const hasChanged = logoFile !== undefined || applicationName !== '';
-    const hasSimulationChanged = !getSimulationPending && !getSimulationError && (isSimulationEnabled !== desiredSimulationState)
-    const simulationLabel = hasSimulationChanged ? this.desiredSimulationLabel[desiredSimulationState] : this.currSimulationLabel[isSimulationEnabled];
+    const hasSimulationChanged = !getSimulationPending
```

Comment:
Why are we reformatting code that has not changed?

Reply:
Coding Standard: Max 90 to 110 chars per line